src/controllers/AssetsController.php

```diff
@@ -822,54 +821,39 @@ public function actionSaveImage(): Response
                 throw new BadRequestHttpException('Invalid cropping parameters passed');
             }
 
+            // TODO Fire an event for any other image editing takers.
+            $transformer = new ImageTransformer();
```

Comment:
@andris-sevcenko Did you forget to add this event?

Reply:
50/50. There are no other image editing takers yet, so I skipped this part.